src/App/Patterns/content/Login/index.js

```diff
@@ -24,7 +24,7 @@ const Overview = () => (
                 The login page is customizable and components can be removed or added to fit a specific products needs.
             </div>
         </div>
-        <ComponentPreview language="html" codeFigure>
+        <EditableComponentPreview language="html" codeFigure>
```

Comment:
Same here, IMO it is nicer to only have one playground pr page 🕺  I am ready to fight for it, and lose.

Reply:
first one is not editable though (the preview is just an image, not the actual output of the code shown. It never was even before those code sandboxes), so I've set the 1st and 2nd as read-only, and set the 3rd/last as interactive
This is very easy to change in the future if we change our mind anyway

Reply:
haha yeah, I know the first is an image :D
One can still edit, even though it is read-only